python/activator/middleware_interface.py

```diff
@@ -663,16 +665,21 @@ def export_outputs(self, visit: Visit, exposure_ids: set[int]) -> None:
         self._export_subset(visit, exposure_ids, "raw",
                             in_collections=self._get_raw_run_name(visit),
                             out_collection=self.instrument.makeDefaultRawIngestRunName())
+
         umbrella = self.instrument.makeCollectionName("prompt-results")
-        self._export_subset(visit, exposure_ids,
-                            # TODO: find a way to merge datasets like *_config
-                            # or *_schema that are duplicated across multiple
-                            # workers.
-                            self._get_safe_dataset_types(self.butler),
-                            self.output_collection + "/*",  # exclude inputs
-                            umbrella)
-        _log.info(f"Pipeline products saved to collection '{umbrella}' for "
-                  f"detector {visit.detector} of {exposure_ids}.")
+        latest_run = self.butler.registry.getCollectionChain(self.output_collection)[0]
```

Comment:
Is this an implementation detail that may get changed in the Rubin middleware? Would it make sense to ask Middleware to implement a public method that gives the latest run?

Reply:
No, `MiddlewareInterface` manages the collections purely internally, which is why I made the run order a class invariant. At present, the invariant is enforced by only adding runs to the chain by calling `_prepend_collection`.

Reply:
Never mind... My questions is answered by [this line ](https://github.com/lsst-dm/prompt_prototype/blob/9b4a6e715b1c3db3868390a6108abef8b3e5883b/python/activator/middleware_interface.py#L135-L136)